src/main/java/org/esi/grable/menus/controller/MenuController.java

```diff
@@ -0,0 +1,39 @@
+package org.esi.grable.menus.controller;
+
+import org.esi.grable.menus.model.Menu;
+import org.esi.grable.menus.service.MenuService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/menus")
```

Comment:
"menus" in here, but "menu" in documentation and Rest file

Reply:
done